examples/multi-kanary/demo.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+DEMO_DIR="$(cd "$(dirname "${0}")" && pwd)"
+CURRENT=$PWD
+
+cd ${DEMO_DIR}/../..
+
+ROOTDIR=$PWD
+
+. script/demo-utils.sh 
+
+function wait_for_deployment() {
+    deployment=${1}
+    desc "waiting for deployment: ${deployment}"
+    until [ $(kubectl get deployment ${deployment} -ojsonpath="{.status.conditions[?(@.type=='Available')].status}") == "True" ] > /dev/null 2>&1; do sleep 1; kubectl get deployment ${deployment}; done
```

Comment:
I saw this on the documentation
```
kubectl wait --for=condition=available --timeout=600s deployment/myapp -n namespace
```

Reply:
I tried it, it works well, still I prefer the "until" function because it allow us to display progress.